chart/templates/postgres.yaml

```diff
@@ -23,6 +23,8 @@ spec:
     metadata:
       labels:
         app: postgresql
+        app.kubernetes.io/name: postgresql
+        app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
Need these labels so the chart-testing tool `ct` shows the logs and describes the pod.